Getting-Started/Setup/Server-Setup/Load-Balancing/azure-web-apps.md

```diff
@@ -0,0 +1,58 @@
+---
+versionFrom: 8.0.0
+---
+
+## Load Balancing Azure Web Apps
+
+Ensure you read the [overview](index.md) before you begin - you will need to ensure that your ASP.NET & logging configurations are correct.
+
+### Azure Requirements
+
+* You will need to setup 2 x Azure Web Apps - one for the master (administrative) environment and another for your front-end environment
+* You will need 1 x SQL server that is shared with these 2 web apps
+
+### Lucene/Examine configuration
+
+The single instance backoffice Web App should be set to use [SyncTempEnvDirectoryFactory](file-system-replication.md#examine-directory-factory-options).
+
+The multi instance front end Web App should be set to use [TempEnvDirectoryFactory](file-system-replication.md#examine-directory-factory-options).
+
+### Umbraco TEMP files
+
+Store the Umbraco temporary files in the local server's 'temp' folder. Achieve this by changing this configuration setting to 'true' in the web.config. The downside is that if you need to view this configuration file you'll have to find it in the temp files. Locating the file this way isn't always clear.
+			
+```xml
+<add key="Umbraco.Core.LocalTempStorage" value="EnvironmentTemp" />
+```
+
+### Umbraco PublishedCache
+
+When Azure Web Apps auto transition between hosts, you scale the instances or you utilise slot swapping you may experience issues with the Umbraco Published Cache becoming locked unless it is configured to ignore the local database. 
+
+A composer is required to configure this option
+
+```csharp
+composition.Register(factory => new PublishedSnapshotServiceOptions
+{
+    IgnoreLocalDb = true
+});
+```
+
+### Steps to set-up a environment
+
+1. Create an Azure SQL database
+2. Install Umbraco on your master environment and ensure to use your Azure SQL Database
+3. Install Umbraco on your front-end environment and ensure to use your Azure SQL Database
```

Comment:
Should we mention the word 'Replica' here? for the front end environment...

front end implies to me a little headless situation, where the actual Umbraco application isn't deployed... but that isn't the case (yet) where in future we might have a decoupled runtime... The ServerRole 'enum' has Master, Replica, Single and Unknown... so I reckon if we start sewing the word Replica in here it will help understanding and debugging for people in the future, can still call it the Replica Front End if that makes sense?

Reply:
Frond servers can be Master rather than Replcia but perhaps I'll update these steps to include fixing Master and Replic....